recommendations/static/recommendations/css/index.css

```diff
@@ -3,26 +3,28 @@
 
 :root {
   --primary-font: "Inter", sans-serif;
-  --heading-color: #092e20;
-}
-
-body {
-  font-family: var(--primary-font);
-  background-color: rgb(255, 255, 255);
-  display: flex;
-  flex-direction: column;
-  gap: 20px;
-  padding: 20px 20px;
+  --heading-color: black;
+  --footer-color: green;
```

Comment:
green?

Reply:
yeah green
i see that it's confusing as we have an actual footer, this is just for the color on the text at the bottom of the page, thus 'footer'.

i'll update this variable to be more descriptive in this pr if there's anything else to be updated. otherwise, i'll do it next time as i don't think it's too big of an issue.